docs/editor-sprite-renderer.en-US.md

```diff
@@ -0,0 +1,45 @@
+---
+order: 8
+title: SpriteRenderer Component
+type: Editor
+---
+
+The sprite renderer component is used to display images in a 3D/2D scene, see [Sprite component](${docs}sprite-renderer-cn) for details.
+
+## Add SpriteRenderer component
+
+When you need to display a image, you need to add a sprite renderer component to an entity first, as follows:
```

Comment:
display an image

Reply:
It's better: When you need to display a image, a sprite renderer component should be added to an entity first.

use more passive words.

Reply:
done